storage/src/metadata/mod.rs

```diff
@@ -32,6 +32,12 @@
 //! not be guaranteed to recover the latest complete state from disk on restart as half of a blob
 //! could be old data and half new data).
 //!
+//! # Efficient Writes
+//!
+//! When an update is committed, only updated bytes are actually written to disk. This makes `Metadata`
```

Comment:
```suggestion
//! When an update is committed, only updated bytes are actually written to disk. This makes [Metadata]
```

We also use backticks above so idk if we want to keep this as is for parallelism or change the other instances.